<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>625</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Угадай Город</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>800</width>
     <height>600</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>800</width>
     <height>600</height>
    </size>
   </property>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>800</width>
      <height>600</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>800</width>
      <height>600</height>
     </size>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="menuLayer">
     <property name="minimumSize">
      <size>
       <width>800</width>
       <height>600</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>800</width>
       <height>600</height>
      </size>
     </property>
     <widget class="QLabel" name="logo">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>331</width>
        <height>131</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image:url(:/util_img/util_img/Logo_pink_grey.png)</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="playButton">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>150</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Играть</string>
      </property>
     </widget>
     <widget class="QPushButton" name="scoreboardButton">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>200</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Таблица лидеров</string>
      </property>
     </widget>
     <widget class="QPushButton" name="exitButton">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>520</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Выход</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="settingsLayer">
     <property name="minimumSize">
      <size>
       <width>800</width>
       <height>600</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>800</width>
       <height>600</height>
      </size>
     </property>
     <widget class="QPushButton" name="BackToMenu">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>500</y>
        <width>171</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Выйти в главное меню</string>
      </property>
     </widget>
     <widget class="QPushButton" name="PlayButton2">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>500</y>
        <width>111</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Играть</string>
      </property>
     </widget>
     <widget class="QLabel" name="TimerL">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>130</y>
        <width>141</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(255, 255, 255, 0);
color: rgb(0, 0, 0);
font: 700 18pt &quot;Sitka Small&quot;;</string>
      </property>
      <property name="text">
       <string>Таймер:</string>
      </property>
     </widget>
     <widget class="QLabel" name="HintsL">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>190</y>
        <width>191</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(255, 255, 255, 0);
color: rgb(0, 0, 0);
font: 700 18pt &quot;Sitka Small&quot;;</string>
      </property>
      <property name="text">
       <string>Подсказки:</string>
      </property>
     </widget>
     <widget class="QLabel" name="PackL">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>250</y>
        <width>191</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(255, 255, 255, 0);
color: rgb(0, 0, 0);
font: 700 18pt &quot;Sitka Small&quot;;</string>
      </property>
      <property name="text">
       <string>Набор:</string>
      </property>
     </widget>
     <widget class="QSlider" name="TimerSlider">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>130</y>
        <width>211</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>4</number>
      </property>
      <property name="singleStep">
       <number>1</number>
      </property>
      <property name="pageStep">
       <number>1</number>
      </property>
      <property name="value">
       <number>1</number>
      </property>
      <property name="tracking">
       <bool>true</bool>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="tickPosition">
       <enum>QSlider::TicksBelow</enum>
      </property>
      <property name="tickInterval">
       <number>1</number>
      </property>
     </widget>
     <widget class="QCheckBox" name="HintsCheck">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>195</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QComboBox" name="PackChoiceBox">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>260</y>
        <width>151</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <item>
       <property name="text">
        <string>Россия</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Мир</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="TimerValueL">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>130</y>
        <width>61</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(255, 255, 255, 0);
color: rgb(0, 0, 0);
font: 700 18pt &quot;Sitka Small&quot;;</string>
      </property>
      <property name="text">
       <string>30</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
